config/samples/consul_v1alpha1_serviceresolver.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceResolver
+metadata:
+  name: serviceresolver-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
It might be nice to flesh out these samples more to make them something that can be applied to the cluster for demos/testing

Reply:
Are we gonna keep these samples around? I'd think we'll want to clean up a bunch of this stuff from this repo

Reply:
oh..i honestly like using these when i want test the webhooks or the reconciler real quick. It's nice to have a sample handy to just `pbpaste | k apply -f -` onto the cluster and voila.

Reply:
👍 I added this as a todo in our list.